test/unit/Cardano/Wallet/BlockSyncerSpec.hs

```diff
@@ -114,7 +117,7 @@ newtype Blocks = Blocks [[Block]]
 instance Arbitrary Blocks where
     -- No Shrinking
     arbitrary = do
-        n <- arbitrary
+        n <- fromIntegral . (`mod` 42) <$> arbitrary @Word8
```

Comment:
why `42` here?

Reply:
To have a hard max bound. It's not more useful to generate list of 255 blocks than 42 blocks in practice. We aren't benchmarking anything here, so, we want the list to be big enough to generate enough entropy in the blocks groups, but not too big to not take 5 minutes to run. 

`42` turned out to be a good enough value :)

Reply:
sounds reasonable. thanks for the explanation :+1: